ConsoleApp/ConsoleApp/Program.cs

```diff
@@ -0,0 +1,38 @@
+﻿int x, y;
+bool successX = true,
+    successY = true;
+string? xInput, yInput;
+
+do
+{
+    Console.WriteLine(successX ? "Please define X variable" : "X should be a number, so please define X variable again");
+    xInput = Console.ReadLine();
+    successX = int.TryParse(xInput, out x);
+}
+while (!successX);
+
+do
+{
+    Console.WriteLine(successY ? "Please define Y variable" : "Y should be a number, so please define Y variable again");
+    yInput = Console.ReadLine();
+    successY = int.TryParse(yInput, out y);
+}
+while (!successY);
+
+int sum = 0;
+
+if (x == y)
```

Comment:
you can do even without this branch. if you remove **if/else** then loop **for** is going to add only one number when x == y